Android-Flutter-Merge.yaml

```diff
@@ -76,33 +79,33 @@ steps:
   - AC_OMIT_ZERO_PATCH_VERSION: $AC_OMIT_ZERO_PATCH_VERSION
   - AC_VERSION_FLAVOR: $AC_VERSION_FLAVOR
   outputs:
-  - AC_ANDROID_NEW_VERSION_CODE: ''
-  - AC_ANDROID_NEW_VERSION_NAME: ''
+  - AC_ANDROID_NEW_VERSION_CODE: AC_ANDROID_NEW_VERSION_CODE
+  - AC_ANDROID_NEW_VERSION_NAME: AC_ANDROID_NEW_VERSION_NAME
 - componentType: appcircle_android_flutter_build
-  componentVersion: "1.0.*"
+  componentVersion: 1.0.*
   stepName: Flutter Build for Android
   enabled: true
   onError: $exit
   webUrl: https://github.com/appcircleio/appcircle-android-flutter-build-component
   repoUrl: https://github.com/appcircleio/appcircle-android-flutter-build-component.git
-  commit: ff1f6a2
+  commit: 09b7012
   inputs:
   - AC_FLUTTER_PROJECT_DIR: $AC_REPOSITORY_DIR
-  - AC_FLUTTER_BUILD_MODE: $AC_FLUTTER_BUILD_MODE 
+  - AC_FLUTTER_BUILD_MODE: $AC_FLUTTER_BUILD_MODE
   - AC_FLUTTER_BUILD_EXTRA_ARGS: $AC_FLUTTER_BUILD_EXTRA_ARGS
   - AC_OUTPUT_TYPE: $AC_OUTPUT_TYPE
   outputs:
   - AC_APK_PATH: AC_APK_PATH
   - AC_AAB_PATH: AC_AAB_PATH
-- componentType: appcircle_android_sign
-  componentVersion: "1.0.*"
+- componentType: appcircle_android_flutter_sign
+  componentVersion: 1.0.*
   stepName: Android Sign
   enabled: true
   onError: $exit
   webUrl: https://github.com/appcircleio/appcircle-android-sign-component
   repoUrl: https://github.com/appcircleio/appcircle-android-sign-component.git
-  commit: a3d10d4
-  inputs:  
+  commit: 0855c87
```

Comment:
This commit number does not match with repo's last commit in master branch.

https://github.com/appcircleio/appcircle-android-sign-component